exec/java-exec/src/test/java/org/apache/drill/exec/record/vector/TestValueVector.java

```diff
@@ -223,14 +231,187 @@ public void testNullableVarLen2() {
     }
   }
 
+  private static DrillBuf combineBuffers(final BufferAllocator allocator, final DrillBuf[] buffers) {
+    // find the total size we'll need
+    int size = 0;
+    for(final DrillBuf buffer : buffers) {
+      size += buffer.readableBytes();
+    }
+
+    // create the new buffer
+    final DrillBuf newBuf = allocator.buffer(size);
+    final DrillBuf writeBuf = newBuf;
+    for(final DrillBuf buffer : buffers) {
+      final DrillBuf readBuf = (DrillBuf) buffer.slice();
+      final int nBytes = readBuf.readableBytes();
```

Comment:
not a show stopper but the whole loop here could be replaced with ByteBuf#readBytes or ByteBuf#getBytes I think.